podman-image/Containerfile

```diff
@@ -48,18 +48,24 @@ RUN if [[ ${PODMAN_PR_NUM} == "" ]]; then \
         rpm-ostree override replace --experimental --freeze \
         --from repo="copr:copr.fedorainfracloud.org:packit:containers-podman-${PODMAN_PR_NUM}" \
         podman; \
-    fi && \
-    rpm-ostree override remove moby-engine containerd runc && \
-    rm -fr /var/cache && \
-    ostree container commit
+    fi
 
 # Install subscription-manager and enable service to refresh certificates
 # Install qemu-user-static for bootc
 # Install gvisor-tap-vsock-gvforwarder for hyperv
+# Install device-mapper (this satisfies the deps for qemu-user-static
 # Install ansible for post-install configuration
-RUN rpm-ostree install subscription-manager gvisor-tap-vsock-gvforwarder qemu-user-static ansible-core && \
+# Remove unwanted packages
+# Remove man pages (man binary is not present)
+RUN dnf install -y --setopt=install_weak_deps=false subscription-manager device-mapper qemu-user-static-aarch64 qemu-user-static-x86
+RUN dnf install -y subscription-manager gvisor-tap-vsock-gvforwarder ansible-core
+
+RUN dnf remove -y moby-engine containerd runc toolbox qed-firmware docker-cli && \
```

Comment:
I think these should all be done in one step so we only have one layer

Reply:
(I say this over and over and over) 
And one way to make that look nice is to use https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/

Reply:
Agree, using heredoc syntax for the entire shell script part would make it much more readable.